### PR TITLE
Updates patch for mysql 8.21

### PR DESCRIPTION
**Pull request by catenacyber**
Should fix build failure https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=24152 with solved conflict in vio.cc

### CODE REVIEW

File: projects/mysql-server/build.sh [hunk 1]
```diff
@@ -27,6 +27,8 @@ else
 fi
 make install
 cp $OUT/mysql/bin/fuzz* $OUT/
+#temporary disabled
```
Comment (inferno-chromium) [inline projects/mysql-server/build.sh:30]: "# FIXME: Temporary disabled due to ...." and end comment with period.

File: projects/mysql-server/targets/fuzz_docommand.cc [hunk 2]
```diff
@@ -40,6 +40,7 @@ extern "C" int LLVMFuzzerInitialize(const int* argc, char*** argv) {
 }
 
 
+//TODO fix with 8.21
```
Comment (inferno-chromium) [inline projects/mysql-server/targets/fuzz_docommand.cc:58]: // FIXME: Fix .... with 8.21
USE FIXME: or TODO(catenacyber): 
and please add what is broken in fix comment.

File: projects/mysql-server/build.sh (hunk 1 above)
Comment (catenacyber) [reply projects/mysql-server/build.sh]: Ok

File: projects/mysql-server/targets/fuzz_docommand.cc (hunk 2 above)
Comment (catenacyber) [reply projects/mysql-server/targets/fuzz_docommand.cc]: Ok

**Comment by catenacyber** [comment]
I improved the comments
Any idea why the CI failed with `fuzz_docommand` which is not meant to be built

File: projects/mysql-server/build.sh [hunk 3]
```diff
@@ -27,6 +27,8 @@ else
 fi
 make install
 cp $OUT/mysql/bin/fuzz* $OUT/
+#FIXME: Temporary disabled due to fix needed in target with mysql 8.21
+#rm $OUT/fuzz_docommand
```
Comment (inferno-chromium) [inline projects/mysql-server/build.sh:31]: I dont understand this, why do we need this here, when the rm is commented out. Can you remove these two lines.

File: projects/mysql-server/targets/fuzz_docommand.cc [hunk 4]
```diff
@@ -40,6 +55,7 @@ extern "C" int LLVMFuzzerInitialize(const int* argc, char*** argv) {
 }
 
 
+//FIXME: Fix this buffer with succesful authenticated connection for mysql 8.21
```
Comment (inferno-chromium) [inline projects/mysql-server/targets/fuzz_docommand.cc:58]: Space between // and FIXME. also nit: end comments with period.

**Pull request merged**